.github/workflows/tests.yml

```diff
@@ -38,10 +38,16 @@ jobs:
             python${{ matrix.python-version }}-dev \
             python${{ matrix.python-version }}-venv \
             python3-pip \
-            g++
+            g++ \
+            libgl1
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install tox tox-gh-actions
+          python -m pip install tox
+      - name: Lint with tox
+        run: tox -e lint
+      - name: Format with tox
+        run: tox -e format
```

Comment:
is `pre-commit` something that could be added in the repo? then we could just run `pre-commit run --all-files`
https://github.com/developmentseed/titiler/blob/master/.github/workflows/ci.yml#L55-L56

Reply:
Very good idea, Added a pre-commit config, @vincentsarago would you suggest removing lint and format completely out of the tox file?

Reply:
It's quite a tricky thing it seems to work with different python version in the CI for running tests and using pre-commit, leaving in the tox format and lint as is in github actions as a sanity check.